### ISSUE TITLE
Is this package suitable for daily backup?

### ISSUE THREAD

**Issue by bahman2000** [fluttercommunity/flutter_workmanager#490]
We need a daily backup at 8:00 AM. This is not a immediate task and it is not a problem for us that the task is executed with a delay. For example, if it is executed at 8:10, there is no problem. We don't want to use the [AlarmManager](https://pub.dev/packages/android_alarm_manager_plus) because of the battery consumption problem. We want to use this package and register a task via registerOneOffTask. Inside the callback we schedule another one off task. We always have to calculate the initial delay according to when we want to run the task. Is this package suitable or we have to use the AlarmManager?

**Comment by mym0404** [comment]
You should check [Background Work Overview](https://developer.android.com/guide/background?hl=en).

**Comment by ened** [comment]
Yes — this package is designed for deferrable background work like a daily sync, and the initial-delay pattern you described is the right approach. Caveats: Android/OEM battery savers and iOS scheduling can delay or skip runs (WorkManager is 'best effort', not a fixed alarm clock); if the backup must run at 08:00 sharp, use AlarmManager/exact alarms instead. See the docs' platform capability matrix. Closing as answered.
